twake/frontend/public/locales/ru.json

```diff
@@ -674,5 +674,9 @@
     "scenes.app.integrations_parameters.company_applications_table.title": "Установлен в вашей компании",
     "app.state.recoil.hooks.use_current_company_applications.toaster_delete" : "Успешно удалил {{$1}} из вашей компании",
     "app.state.recoil.hooks.use_current_company_applications.toaster_add" : "{{$1}} успешно добавлен в вашу компанию",
-    "scenes.apps.messages.input.replied_to" : "ответил "
+    "scenes.apps.messages.input.replied_to" : "ответил ",
```

Comment:
@romanesko could you check these translations please ? 😄

Reply:
One is wrong.
The correct form: 

"components.rich_text_editor.plugins.suggestions.default_message.display_results": "Введите минимум три символа для отображения пользователей"

By the way, does it have to be letters? I suppose it can be any characters

Reply:
@stephanevieira75 you handle this before vacations or not ? :)

Reply:
Updated 😄 
@romanesko I'm not sure about this, WDYT @RomaricMourgues ?

Reply:
Anything but space and ponctuations here.
Thanks Stephane and happy Christmas !